lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_slack_messages.rb

```diff
@@ -0,0 +1,30 @@
+require 'rspec/expectations'
+RSpec::Matchers.define :respond_with_slack_messages do |expected|
+  match do |actual|
+    raise ArgumentError, 'respond_with_slack_messages expects an array of ordered responses' unless expected.is_a? Array
```

Comment:
I would check for `Enumerable` instead or maybe just that it responds to `each`? This way I can get the messages from some collection, database, whatever, if needed.